### PR TITLE
Move promoted blog content into collection with Netlify

### PR DESCRIPTION
**Pull request by lauraGgit**
Fixes issue(s) # .

[CircleCI](https://circleci.com/gh/18F/18f.gsa.gov/tree/BRANCH_NAME)

[:sunglasses: PREVIEW](https://federalist-c58f58dc-a215-4616-a54c-12b5eb011096.app.cloud.gov/preview/18f/18f.gsa.gov/BRANCH_NAME/)

[Preview README for this branch](https://github.com/18F/18f.gsa.gov/blob/BRANCH_NAME/README.md)

Changes proposed in this pull request:
-
-
-

Checklist:
- [ ] All images being added have been optimized **_--> [learn more](https://18f.gsa.gov/styleguide/images) about how to optimize images <--_**

/cc @relevant-people

### CODE REVIEW

File: blog/index.html [hunk 1]
```diff
@@ -48,54 +48,26 @@ <h2>{{ page.blog_tagline }}</h2>
...
-        <li>
-          <a href="{{ site.baseurl }}/tags/lessons-learned">Lessons Learned</a>
+          <a href="{{ site.baseurl }}/tags/{{ item.tag }}">{{ item.topic }}</a>
         </li>
+        {% endfor %}
       </ul>
       <div class="sidebar-callout">
         <h4>Most popular</h4>
         <ul>
+          {% for item in featured_blog_content.popular_posts %}
```
Comment (apburnes) [inline blog/index.html:64]:
```suggestion
          {% for item in featured_blog_content.promoted_posts %}
```

**Review by apburnes — COMMENTED** [review]
Just need to fix a key to get promoted posts to render. 👍

File: blog/index.html (hunk 1 above)
Comment (Dahianna) [reply blog/index.html:64]: @lauraGgit quick suggestion from @apburnes

**Comment by lauraGgit** [comment]
@apburnes thanks for the fix, does added it in. That mean this is good to go?

**Comment by Dahianna** [comment]
Preview looks good to me @apburnes

**Pull request merged**